src/NuGet.Services.AzureSearch/ICommand.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace NuGet.Services.AzureSearch
+{
+    public interface ICommand
```

Comment:
nit: Do you think it would be better if this was `IAzureSearchCommand` instead of just `ICommand`? I thought this was [System.Windows.Input.ICommand](https://docs.microsoft.com/en-us/dotnet/api/system.windows.input.icommand?view=netframework-4.8) at first.

Reply:
Sure, sounds good!